src/lib/components/LibrarySection.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts">
+  export let section;
+</script>
+
+<div class="section-outer">
+  <h2 class="title">{section.title}</h2>
+  <p class="subtitle">{section.playlists.length} Playlists</p>
+  <div class="section">
+    {#each section.playlists as playlist}
+      <div class="library-card">
+        <a href="/">
+          <img src={playlist.imagePath} alt="Playlist Preview" />
+        </a>
+        <h2 class="title">{playlist.name}</h2>
+        <!-- TODO: Pass this playlist into the next page -->
+        <a href="/library/playlists/{playlist.slug}" class="btn btn-blurple">More Info</a>
+      </div>
+    {/each}
```

Comment:
Okay so when I was scrolling horizontally just holding the mouse wheel it felt noticeably choppy. Maybe adding a `key` to each element will help, but I doubt it (we should do it anyway though). It's also a bit annoying if I click on one towards the end and then come back, cause I have to find it again. Should we maybe just consider pagination here instead?